### PR TITLE
testing of dash.test.js

### PR DESCRIPTION
**Pull request by siddharthp538**
## Proposed changes

 This PR consists of adding testing of dash.test.js 

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] refactor
- [ ] documentation
- [ ] other

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

Are tests included?
- [x] yes (*bugfixes and features will not be merged without tests*)
- [ ] no

Breaking Changes?
- [ ] yes (*breaking changes will not be merged unless absolutely necessary*)
- [x] no

List any relevant issue numbers:

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,7 +70,8 @@
     "dayjs": "^1.8.13",
     "lodash": "^4.17.11",
     "path": "^0.12.7",
-    "uuid": "^3.3.2"
+    "uuid": "^3.3.2",
+    "yarn": "^1.16.0"
```
Comment (atherdon) [inline package.json]: i'm not sure if we need yarn here as module. at my PC yarn installed globally

Comment (siddharthp538) [reply package.json]: Yeah, not necessary. Just installed to remove the error of npm. Should I uninstall?

**Comment by atherdon** [comment]
add in next time issue number here #22 tests looks good

**Comment by siddharthp538** [comment]
Alright!

File: package.json (hunk 1 above)
Comment (atherdon) [reply package.json]: yes please

**Pull request merged**